scico/_python37.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2021-2022 by SCICO Developers
+# All rights reserved. BSD 3-clause License.
+# This file is part of the SCICO package. Details of the copyright and
+# user license can be found in the 'LICENSE' file distributed with the
+# package.
+
+""" Alter the behavior of (monkey patch) several modules so that SCICO
+is compatible with Python 3.7, allowing it to run on Google Colab.
+This file should be removed when/if Colab upgrades to Python 3.8.
+"""
+
+import sys
+
+if sys.version_info.minor == 7:
```

Comment:
Perhaps safer to also check that major version is 3 to avoid incorrect logic when 4.7 comes around?